projects/storefrontstyles/scss/theme/sparta/_fonts.scss

```diff
@@ -1 +1,4 @@
-@import url('#{$font-url}');
+@import '@fontsource/open-sans/300.css';
```

Comment:
added 4 files of family 'open-sans' in weight: 300,400,600,700 to mirror the font that were previously downloaded on Google Fonts: https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700